src/components/common/Drawer.tsx

```
@@ -32,22 +31,14 @@ export default function Drawer({ children, isOpen, setIsOpen }: DrawerProps) {
                 setIsOpen(false);
               }}
-            >
-              <Image
-                src={Cancel}
-                width={24}
-                height={24}
-                alt="cancel"
-                className="h-5"
-              />
-            </button>
+            ></button>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)